config/addon/addon-config.yaml

```diff
@@ -0,0 +1,167 @@
+# Copyright 2020 The Kubermatic Kubernetes Platform contributors.
```

Comment:
Do we also want to create an `AddonConfig` for cert-manger plugin?

Reply:
yup, good idea